src/applications/disability-benefits/996/tests/validations/issues.unit.spec.js

```diff
@@ -136,7 +136,8 @@ describe('areaOfDisagreementRequired', () => {
   it('should show an error with other selected, but no entry text', () => {
```

Comment:
Should the name of this test be updated since the 'other' option is no longer a selectable option?

Reply:
Fixed, thanks!